### ISSUE TITLE
Utilize community.sap_install collection

### ISSUE THREAD

**Issue by Klaas-** [Azure/sap-automation#61]
**Is your feature request related to a problem? Please describe.**
I think it would be a good idea to utilize the SAP Ansible collection instead of maintaining own ansible code
https://github.com/sap-linuxlab/community.sap_install
(Or merge your roles into the SAP roles)

Greetings
Klaas

**Comment by Klaas-** [comment]
@KimForss it also seems the sap linuxlab will host terraform modules for infrastructure creation, maybe you can look into merging your work into their tf code. https://github.com/sap-linuxlab/sap-linuxlab.github.io/issues/1

**Comment by KimForss** [comment]
Hi @Klaas We will look into it when we start the planning for the next phase - at the moment restructuring the code base would risk some of the deliverables we have committed to.

**Comment by PankajMeshramCSA** [comment]
@KimForss @Klaas- - Could you please confirm if this can be closed?

**Comment by klaas** [comment]
@PankajMeshramCSA Can't confirm. I guess you meant @Klaas-

**Comment by Klaas-** [comment]
@PankajMeshramCSA no, you're still using the non-standard ansible roles you develop yourself, otherwise you'd include the roles from the sap-linuxlab here https://github.com/Azure/sap-automation/blob/main/deploy/ansible/requirements.yml

**Comment by PmeshramPM** [comment]
Hi @Klaas-  Thank you for the thoughtful suggestion and for highlighting the potential of the community.sap_install Ansible collection. We truly appreciate the effort and collaboration from the community around SAP automation tooling.
We’ve reviewed the proposal to integrate the community.sap_install Ansible collection into SDAF. After careful consideration, we’ve decided not to proceed at this time due to the following:
1. Architectural Alignment: SDAF is tightly integrated with Terraform + Ansible for Azure-specific deployments. The community collection follows a different structure, requiring significant refactoring.
2. Azure-Specific Features: SDAF includes native Azure integrations (e.g., Monitor, Key Vault, Storage Accounts, Azure Files, and Shared Disks etc.) not fully supported in the community roles.
3. Version Control: SDAF maintains dependency management catering to various reference architectures published to our documentation, which would be impacted by external collections with independent lifecycles.
We remain open to future collaboration where alignment can be achieved without compromising SDAF’s core design.
